### DISCUSSION TITLE
Questions from the February 8, 2024 sandbox webinar

### DISCUSSION THREAD

**Discussion by dadahl** [open-voice-interoperability/open-voice-sandbox#2] [Q&A]
Thanks to everyone who attended the sandbox webinar on February 8! 
We had some questions in the Zoom chat that we didn't have time to discuss, and for the ones that we did have time to discuss, I'll respond again here for the record. 

 Question: OpenAI account type required - pro or free ?
Answer: No OpenAI account is required unless you want to include an OpenAI LLM in the set of sandbox assistants. 

Question: Able to search on log file content now - or a future feature?
Answer: The log file display is just a web page, so you can search it like any web page. We are considering more specific filtered searches for the future.  

Question: Is this system multilingual?
Answer: Yes, but that is a feature of specific assistants, not the sandbox or the specifications. As the standards evolve, we envision that there will be ways to make it easier to find assistants with specific language capabilities and/or to include translation assistants in the conversation.

Question: Why didn't the smart library hand the pizza order  over to a pizza ordering programme? 
Answer: The smart library is an AI agent specialized in information about books, so it is designed to not provide information out of that scope.
Question: wouldn't discovery kick in?
Answer: The Smart Library could have been designed to discover another assistant that could find a pizza restaurant and hand off the conversation to that assistant, but that's not how it was designed. It's important for the standards to support both kinds of assistants--ones that just do specific jobs (like the Smart Library) and others that can also find different assistants when they themselves can't handle a task.

Question: Who should we contact for dev questions ?
Answer: 	For dev questions, you should ask your questions in the GitHub Discussions forums, like this one.

**Comment by industrialpoet** [comment]
Question: OpenAI account type required - pro or free ?
Answer: OpenAI is not required, but we are exploring other free open source (even locally executed) LLMs for future releases.
